style.css

```diff
@@ -651,3 +597,119 @@ textarea {
   display: flex;
   margin-top: 25px;
 }
+
+nav {
+  color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav .mainMenu {
+  display: flex;
+  list-style: none;
+  margin-top: -8px;
+}
+
+nav .mainMenu li a {
+  font-family: "Roboto", Arial, sans-serif;
+  display: inline-block;
+  padding: 15px;
+  text-decoration: none;
+  color: #fff;
+  font-size: 16px;
+}
+
+nav .openMenu {
+  font-size: 2rem;
+  margin: 20px;
+  display: none;
+  cursor: pointer;
+}
+
+nav .mainMenu .closeMenu,
+i {
+  font-size: 25px;
+  display: none;
+  cursor: pointer;
+}
+
+nav .logo {
+  margin: 6px;
+  font-size: 18px;
+  cursor: pointer;
+}
+
+.fa-times:hover {
+  color: #fff;
+}
+
+.fa-bars:hover {
+  color: #fff;
+}
+
+@media (max-width: 767px) {
+  nav .mainMenu {
+    height: 101.5vh;
+    position: fixed;
+    font-size: 55px;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    z-index: 10;
+    flex-direction: column;
+    justify-content: center;
+    gap: 15px;
+    background-color: #191765;
+    transition: top 1s ease;
+    background-image: url('images/shapes-mobile-menu.svg');
+    background-repeat: no-repeat;
+    background-size: 100%;
+    display: none;
+  }
+
+  #envelope {
+    display: none;
+  }
+
+  .navLink {
+    font-size: 40px !important;
+  }
+
```

Comment:
- [ ] Good job on the design but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2925) design as much as possible. Please reduce the top margin of your mobile menu, Use the correct font size, and font-weight and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/161624074-77af5f09-7cfb-406c-abc6-45c30d8fcb87.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161623996-d00378d9-b07e-4663-8c12-f01dcd3c3612.png)